Jenkinsfile

```
@@ -41,8 +41,11 @@ for (int i = 0; i < platforms.size(); ++i) {
                     junit '**/target/surefire-reports/TEST-*.xml'
 
                     if (label == 'linux') {
-                      archiveArtifacts artifacts: '**/target/**/*.jar'
-                      findbugs pattern: '**/target/findbugsXml.xml'
+                        archiveArtifacts artifacts: '**/target/**/*.jar'
+                        recordIssues(
+                            enabledForFailure: true, aggregatingResults: true,
+                            tools: [java(), findBugs(pattern: '**/target/findbugsXml.xml')]
```

```suggestion
                            tools: [java(), findBugs(pattern: '**/target/spotbugsXml.xml')]
```

Not sure what is currently used, but we need to upgrade parent POM anyway

---

I tried that earlier with `[SpotBugs] [-ERROR-] No files found for pattern '**/target/spotbugsXml.xml'. Configuration error?`
as the error. Tried locally and it does not generate a spotbugs xml.

I've pushed the parent pom up to `1.54` this is the last version before findsecbugs which adds warnings and breaks the build. But this version will give us spotbugs so we can migrate to spotbugs before fixing all the findsecbugs in a followup.

WDYT?